raiden_contracts/tests/unit/test_unit_signatures.py

```diff
@@ -130,38 +131,121 @@ def test_recover_address_from_balance_proof_update(
         channel_identifier, A, other_token_network=other_token_network
     )
     balance_proof_update_signature = create_balance_proof_countersignature(
-        B, channel_identifier, *balance_proof, other_token_network=other_token_network
+        B,
+        channel_identifier,
+        MessageTypeId.BALANCE_PROOF_UPDATE,
+        *balance_proof,
+        other_token_network=other_token_network,
```

Comment:
why just use one kwarg?

Reply:
It was like this before the PR.

I can go all kwarg.  Then I have to unpack `balance_proof` (but I did so somewhere else).

I can go all positional, but that's a bit worse than the current situation.

Reply:
I'm going all kwarg here.  Eventually `balance_proof` here must become a dictionary, so `**balance_proof` can be used.

Reply:
To be tracked in https://github.com/raiden-network/raiden-contracts/issues/1167